### ISSUE TITLE
Errors even in very simple example

### ISSUE THREAD

**Issue by philwinder** [microsoft/pyright#297]
Hi there. Great work on the plugin.

But I can't seem to get pyright working properly even in simple situations. It struggles to find packages and methods from libraries in a virtual environment. I've tried many different setting combinations (with vscode, without, with pyright settings, without, with project venv, external venv, etc.) All seem to fail.

**To Reproduce**
Here I'm quickly creating a new project, virtual env and running it. Then I run pyright at the end. It should not report errors, but it does. I realise that I could customise all the settings, but from what I have read, I believe this should work out of the box. This also occurs in vscode.

```bash
export DIR=$HOME
mkdir -p $DIR/tmp/test
python3 -m venv $DIR/tmp/testvenv ; source $DIR/tmp/testvenv/bin/activate
pip install numpy
cd $DIR/tmp/test
printf "import numpy\nprint(numpy.ones(1))\n" > main.py
python main.py
pyright  --verbose --venv-path=$DIR/tmp/testvenv
```

This is what I see at the output:
```
Collecting numpy
  Using cached https://files.pythonhosted.org/packages/ea/f4/acaa005b20777fc56a1dc0cae228ab2cb5a7f09a7e7fcb6d4619ce24a1b7/numpy-1.17.3-cp37-cp37m-macosx_10_9_x86_64.whl
Installing collected packages: numpy
Successfully installed numpy-1.17.3
You are using pip version 19.0.3, however version 19.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
[1.]
No configuration file found.
Searching for source files
Found 1 source file
/Users/phil/tmp/test/main.py
  2:13 - error: 'ones' is not a known member of module
1 error, 0 warnings
Completed in 2.274sec
```

**Expected behaviour**
No errors

**VS Code extension or command-line**
Both

**Additional context**
Python 3.7.4 (default, Sep  7 2019, 18:27:02)
MacBook Pro (Retina, 13-inch, Mid 2014)
OSX Mojave 10.14.6 
pyright 1.0.78
Node v12.11.1
vscode  1.39.2

Thanks, Phil

**Comment by erictraut** [comment]
Thanks for the bug report. This same issue was reported by a couple of people. It's due to a regression that I introduced in the last version. I'll have it fixed by the end of the day. Sorry for any inconvenience. 

Incidentally, the problem doesn't occur if you have type stubs for the libraries that you're importing.

**Comment by erictraut** [comment]
I just published 1.0.79, which addresses this problem. Thanks again for reporting it.
